design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -247,8 +242,8 @@ type NegotiableQuoteHistoryChanges {
 
 # Usage in Luma: https://github.com/magento/magento2b2b/blob/0e791b5f7cd604ee6ee40b7225807c01b7f70cf2/app/code/Magento/NegotiableQuote/view/base/templates/quote/history.phtml#L40-L73
 type NegotiableQuoteHistoryStatusChange {
-    old_status: NegotiableQuoteStatus @doc(description: "Will be null for the first history entry on a Negotiable Quote")
-    new_status: NegotiableQuoteStatus!
+    old_status: String @doc(description: "Will be null for the first history entry on a Negotiable Quote")
```

Comment:
Can you help me understand why changing this to `String` fixes the issue? `old_status` set to `NegotiableQuoteStatus` can still be `null`

Reply:
Andrew it seems graphql spec doesn't allow null for enum, can we add a extra value for enum instead of string

Reply:
https://spec.graphql.org/June2018/#sec-Enum-Value reference

Reply:
> it seems graphql spec doesn't allow null for enum

`enum` is a specific type, but all fields, by default, can be `nullable`. The type assigned to a field shouldn't impact it, unless the type is wrapped in a Non-Null wrapper

Reply:
Are you hitting errors implementing that? If so, please provide the details here - could be a webonyx bug.

Reply:
yes, @RakeshJesadiya please share the error thrown.

Reply:
Looks like @rakeshjesadiya got it figured out https://github.com/magento/architecture/pull/422#issuecomment-675407366